### ISSUE TITLE
This Works on GLPI 9.3.1?

### ISSUE THREAD

**Issue by joaotome1994** [JulioAugustoS/glpiorderservice#12]
Hello Julio!

GLPI Order Service Plugin works on GLPI 9.3.1? I put the extracted zip on \var\www\html\glpi\plugins 
but in Plugins dont appear Order Service. 
Can u help?

**Comment by JulioAugustoS** [comment]
Hello Friend,
you have downloaded the available version of the utility, when you put the plugin in the plugins folder of your GLPI the folder name of the plugin must be orderservice, make sure it is exactly like this and anything let me know that I will be happy to help friend.

**Comment by joaotome1994** [comment]
yes. Folder is named orderservice but dont appear in plugins on GLPI 9.3.1 .  Can u test please?

**Comment by joaotome1994** [comment]
![glpi](https://user-images.githubusercontent.com/23320203/46618610-e5822c80-cb17-11e8-8039-dd88c6aa9215.png)

See the plugin dont appear.
orderservice is folder extracted glpi-orderservice-2.0.0.zip

Can u help me please?

**Comment by JulioAugustoS** [comment]
Yes my friend, I'll give you a test and I'll give you some feedback.
